code/modules/admin/verbs/debug.dm

```diff
@@ -664,9 +664,10 @@ GLOBAL_PROTECT(LastAdminCalledProc)
 		if ("Custom")
 			//use custom one
 			M.equipOutfit(custom)
+			M.dna.species.after_equip_job(src, M)
 		else
 			M.equipOutfit(dresscode)
-
+			M.dna.species.after_equip_job(src, M)
```

Comment:
This was a pretty quick fix, but I'm sure I'm overlooking something bad that could come from this.

I have to alter a few loadouts, because some of them are a wee bit broken - the detective spawning with a cigarette stops a mask from being added. Leading to the death of detective voxxy.